src/renderer/components/ft-prompt/ft-prompt.vue

```diff
@@ -1,11 +1,19 @@
 <template>
   <div
     class="prompt"
+    tabindex="-1"
     @click="handleHide"
+    @keydown.enter="handleHide"
   >
-    <ft-card class="promptCard">
+    <ft-card
+      class="promptCard"
+      :aria-labelledby="'dialog-' + sanitizeForHtmlId(label)"
```

Comment:
you could make `sanitizeForHtmlId(label)` a computed property as it's used 4 times in this component